lib/IDE/CodeCompletion.cpp

```diff
@@ -4644,6 +4644,15 @@ static void addExprKeywords(CodeCompletionResultSink &Sink) {
   AddKeyword("#dsohandle", "UnsafeMutablePointer<Void>", CodeCompletionKeywordKind::pound_dsohandle);
 }
 
+static void addAnyTypeKeyword(CodeCompletionResultSink &Sink) {
+  CodeCompletionResultBuilder Builder(
+      Sink, CodeCompletionResult::ResultKind::Keyword,
+      SemanticContextKind::None, {});
+  // pretend 'Any' is from another module
+  Builder.setKeywordKind(CodeCompletionKeywordKind::None);
+  Builder.addTextChunk("Any");
```

Comment:
@benlangmuir, is this the right way to add Any as a keyword? "from another module" doesn't sound right.

@joewillsher, a nitpick: please format comments as English sentences, i.e. `// Pretend 'Any' is from another module.`.